api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -225,21 +224,38 @@ public void sendBetaAccessCompleteEmail(final String contactEmail, final String
   private ImmutableMap<EmailSubstitutionField, String> freeTierDollarThresholdSubstitutionMap(
       final DbUser user, double currentUsage, double remainingBalance) {
 
+    final String freeCreditsResolution =
+        workbenchConfigProvider.get().featureFlags.enableBillingUpgrade
+            ? "you can request for an extension of free credits by contacting support "
```

Comment:
The text is slightly different between threshold alerting and expiration (100% threshold) alerting.  this was already present - I'll confirm w/ Karthik whether this is desired.